src/bank.rs

```diff
@@ -1347,16 +1397,19 @@ mod tests {
     }
 
     #[test]
-    fn test_process_ledger() {
+    fn test_process_ledger_simple() {
```

Comment:
Why add the suffix `_simple` here? I don't see variations of this test. Perhaps you meant `test_process_sample_ledger` since it tests the `sample_ledger` defined in the function above?

Reply:
@garious its hard to grep for it, since its a prefix for other tests